Sources/HipMobileUI/Pages/ExhibitDetailsPage.xaml.cs

```diff
@@ -80,6 +80,11 @@ private void ViewModelOnPropertyChanged(object sender, PropertyChangedEventArgs
             {
                 UpdateAudioBarVisibility();
             }
+
+            if (propertyChangedEventArgs.PropertyName.Equals(nameof(ViewModel.HasAdditionalInformation)))
```

Comment:
Note for reviewers: Even though `IsVisible` is set to `{Binding HasAdditionalInformation}` in `ExhibitDetailsPage.xaml`, this seems to be required to have the setter of `IsVisible` be invoked.